mne/realtime/epochs.py

```diff
@@ -208,6 +211,76 @@ def events(self):
         """The events associated with the epochs currently in the queue."""
         return np.array(self._events)
 
+    @events.setter
+    def events(self, new_events):
+        """Update the internal event list"""
+        self._events = [tuple(new_events[i, :]) for i in range(len(new_events))]
+
+    @property
+    def selection(self):
+        """The current epoch selection"""
+        return np.asarray(self._selection, dtype=int)
+
+    @selection.setter
+    def selection(self, new_selection):
+        """Update the internal selection list"""
+        self._selection = list(new_selection)
+
+    def __len__(self):
+        """Return the number of available epochs"""
+        # we override this since the check for dropped epochs doesn't apply here
+        return len(self._events)
+
+    def _getitem(self, item, reason='IGNORED', copy=True, drop_event_id=True):
+        """Wrap __getitem__ with more options."""
```

Comment:
Is there a way not to copy the code? Maybe by adding an extra option to the existing `__getitem__` or calling `super` here?